src/components/HumanInterface/HumanBack.vue

```diff
@@ -467,6 +474,50 @@ export default defineComponent({
   setup() {
     return { toggleList, chooseMuscle };
   },
+  computed: {
+    neckStyle: function () {
```

Comment:
instead of writing out all those functions consider using inline class bindings in object syntax instead: https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax
e.g.:
`:class="{ neck: selectedMuscle.value == 'neck' }"`
to use selectedMuscle in the template it has to be added to the setup function